units/en/unit7/multi-agent-setting.mdx

```diff
@@ -0,0 +1,57 @@
+# Designing Multi-Agents systems
+
+For this section, you're going to watch this excellent introduction to multi-agents made by <a href="https://www.youtube.com/channel/UCq0imsn84ShAe9PBOFnoIrg"> Brian Douglas </a>.
+
+<Youtube id="qgb0gyrpiGk" />
+
+
+In this video, Brian talked about how to design multi-agents systems. Especially he took a vacuum cleaner multi-agents setting example and asked how they **can cooperate each other**?
+
+To design this multi-agents reinforcement learning system (MARL), we have two solutions.
+
+## Decentralized system
+
+<figure>
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit10/decentralized.png" alt="Decentralized"/>
+<figcaption>
+Source: <a href="https://www.youtube.com/watch?v=qgb0gyrpiGk"> Introduction to Multi-Agent Reinforcement Learning </a>
+</figcaption>
+</figure>
+
+In decentralized learning, **each agent is trained independently from others**. In the example given each vacuum learns to clean as much place it can **without caring about what other vacuums (agents) are doing**.
```

Comment:
```suggestion
In decentralized learning, **each agent is trained independently from others**. In the example given, each vacuum learns to clean as many places as it can **without caring about what other vacuums (agents) are doing**.
```